src/plugins/intel_cpu/src/nodes/mkldnn_roi_align_node.cpp

```diff
@@ -15,14 +15,347 @@
 #include <mkldnn_selective_build.h>
 #include <ngraph/opsets/opset3.hpp>
 
+#include <cpu/x64/jit_generator.hpp>
```

Comment:
[random place]. As far as I can see current impl doesn't provide good enough performance for target scenarios. We need to find a way how to significantly improve it. 
The implementation always utilizes gather approach and store single element per each iteration. It sounds like a suitable approach for nchw layout, but it seems to be inefficient for nspc or blocked layouts. 
Looking on the pool() impl I can see that in case of nspc and blcoked layouts we can load 4 src vectors (part1, part2, ...) by pure vector load cause all elements are placed successively. After all computations the same logic for storage: dst elements are placed successively.
This approach sounds reasonable by semantic of the operations: each channel is processed independently.
Do you have any concerns?

Reply:
@dmitry-gorokhov done! thanks Dmitry.